lib/tasks/read-config.js

```diff
@@ -20,7 +20,7 @@ module.exports = Task.extend({
 
     this.deployConfigPath  = this.deployConfigPath || 'config/deploy.js';
 
-    if (!existsSync(this.deployConfigPath)) {
+    if (!existsSync(path.join(this.root, this.deployConfigPath))) {
```

Comment:
Since [below](https://github.com/ember-cli/ember-cli-deploy/blob/0706b90ab00771d1382835fe7e67e0a95a48e619/lib/tasks/read-config.js#L47) the root is joined to the config path, this seemed like a sane change.